vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -113,13 +106,20 @@ instances:
     #
     # collection_level: 1
 
-    ## @param collect_realtime_only - boolean - optional - default: true
-    ## Set to true to collect only host and VM metrics. Set to false to collect
-    ## cluster, datastore and datacenter metrics as well.
-    ## Collecting non realtime metrics will decrease performance as it
-    ## requires computation from the vCenter server.
```

Comment:
We should probably keep that documentation for the time being, no? Maybe add a note to deprecate it?

Reply:
You're right let's keep it with a deprecation notice.